### DISCUSSION TITLE
Entities translation

### DISCUSSION THREAD

**Discussion by thouel** [saniho/apiEnedis#194] [Q&A]
Hello there, 

First of all, thanks for the contribution, it's a real pleasure to be able to gather those data in HomeAssistant !

I'm using it since a few days and i must confess that i'm a bit lost with the entities provided. 

Could you please confirm i guessed what they hold properly ?
* sensor.myenedis_energy_hours_<prm_id> : energy consumption during the last hour
* sensor.myenedis_energy_hours_cost_<prm_id> : cost of the energy consumption during the last hour
* sensor.myenedis_history_<prm_id>_hc : off peak energy consumption since the last meter reset
* sensor.myenedis_history_<prm_id>_hp : peak energy consumption since the last meter reset
* sensor.myenedis_history_<prm_id>_all : off-peak + peak energy consumption since the last meter reset
* sensor.myenedis_cost_yesterday_<prm_id> : cost of the yesterday's total (peak and off-peak) energy consumption
* sensor.myenedis_<prm_id> : ??
* sensor.myenedis_energy_<prm_id> : ??

PS : i did not find this answer in documentation, yet, in case i missed something, feel free to give me the link, i'll read it carefully :)

See you soon

**Comment by superjaegermaster** [comment]
Hi I just installed it yesterday but for sure can help only on those 2 entities :
sensor.myenedis_history_<prm_id>_hc : how many kwh consumed in "reduced price period" a.k.a Heures Creuse (thus the hc)
sensor.myenedis_history_<prm_id>_hp : how many kwh consumed in "standard price period" a.k.a Heures Pleines  (thus the hp)

These are relevant in case of HC/HP subscription to your energy provider, some people are on HP/HC subscription (intersting if you have an electric car for ex) and some others have  HP all time
